README.md

```diff
@@ -72,6 +72,10 @@ const server = new ProxyChain.Server({
             // If "requestAuthentication" is true, you can use the following property
             // to define a custom error message to return to the client instead of the default "Proxy credentials required"
             failMsg: 'Bad username or password, please try again.',
+
+            // Optional metadata that will be passed back via
+            // `tunnelConnectResponded` or `tunnelConnectFailed` events
+            metadata: { userId: '123' },
```

Comment:
sorry, just one last note 🙈 wouldn't it make more sense to call this `tag` or `customTag` rather than `metadata`?  metadata has a specific meaning. Your call @jirimoravcik

Reply:
Good point, I went with `customTag`, smaller chance of a meaning conflict due to the name